docker-compose.yaml

```diff
@@ -1,21 +1,20 @@
 version: '3.3'
 services:
   postgres:
-   image: postgres:13
-   environment:
-    - POSTGRES_DB=${DB_NAME}
-    - POSTGRES_USER=${DB_USER}
-    - POSTGRES_PASSWORD=${DB_PASSWORD}
-   ports:
-    - 5432:5432
-   volumes:
-    - ./data:/var/lib/postgresql/data
-  
+    image: postgres:13
+    environment:
+      - POSTGRES_DB=${DB_NAME}
+      - POSTGRES_USER=${DB_USER}
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+    ports:
+      - 5432:5432
+    volumes:
+      - ./data:/var/lib/postgresql/data
+
   pgadmin:
     image: dpage/pgadmin4
     environment:
       - PGADMIN_DEFAULT_EMAIL=mrivas@mail.com
```

Comment:
Coordinate with @MarkRiv to remove these data from this file

- [ ] PGADMIN_DEFAULT_EMAIL
- [ ] PGADMIN_DEFAULT_PASSWORD